src/services/completions.ts

```diff
@@ -1143,10 +1143,10 @@ namespace ts.Completions {
                 }
             }
 
-            if (isMetaProperty(node) && (node.keywordToken === SyntaxKind.NewKeyword || node.keywordToken === SyntaxKind.ImportKeyword)) {
-                const completion = (node.keywordToken === SyntaxKind.NewKeyword) ? "target" : "meta";
-                symbols.push(typeChecker.createSymbol(SymbolFlags.Property, escapeLeadingUnderscores(completion)));
-                return;
+            if (isMetaProperty(node) && (node.keywordToken === SyntaxKind.NewKeyword || node.keywordToken === SyntaxKind.ImportKeyword) && contextToken === node.getChildAt(1)) {
+                    const completion = (node.keywordToken === SyntaxKind.NewKeyword) ? "target" : "meta";

```

Comment:
Please correct the formatting. That should make it good to go.

Reply:
Done! Thanks for reviewing this :)